pkg/armhelpers/mockclients.go

```diff
@@ -265,3 +266,26 @@ func (mc *MockACSEngineClient) CreateRoleAssignment(scope string, roleAssignment
 func (mc *MockACSEngineClient) CreateRoleAssignmentSimple(applicationID, roleID string) error {
 	return nil
 }
+
+// DeleteManagedDisk is a wrapper around disksClient.Delete
+func (mc *MockACSEngineClient) DeleteManagedDisk(resourceGroupName string, diskName string, cancel <-chan struct{}) (<-chan disk.OperationStatusResponse, <-chan error) {
+	errChan := make(chan error)
+	respChan := make(chan disk.OperationStatusResponse)
+	go func() {
+		defer func() {
+			close(errChan)
+		}()
+		defer func() {
+			close(respChan)
+		}()
+		errChan <- nil
+		respChan <- disk.OperationStatusResponse{}
+		time.Sleep(1 * time.Second)
```

Comment:
Can we remove this? I think I have a commit in one of the PRs I just merged that removes the sleeps from this file. AFAICT there's no need for them, and removing it just sped up the tests a bit.